### ISSUE TITLE
Request for verifying my steps for training

### ISSUE THREAD

**Issue by gd1925** [HSE-asavchenko/HSE_FaceRec_tf#4]
Hi, 

I have been working on trying to understand the steps for training and obtaining my model for - [age_gender_tf2_224_deep-03-0.13-0.97.pb](https://github.com/HSE-asavchenko/HSE_FaceRec_tf/blob/master/age_gender_identity/age_gender_tf2_224_deep-03-0.13-0.97.pb)
I followed a previously asked question - https://github.com/HSE-asavchenko/HSE_FaceRec_tf/issues/2#issue-450625649
mentioning the two scripts -  facerec_keras_train.py and age_gender_train.py. As per my understanding, the following two step training process is required :  

1. Training the baseline model with facerec_keras_train.py:
- Download the VGGFace2 dataset.
- Run the Training Script - Execute facerec_keras_train.py to start the training on VGGFace2.
- (I presume this will give me - [vgg2_mobilenet.pb](https://github.com/HSE-asavchenko/HSE_FaceRec_tf/blob/master/models/vgg2_mobilenet.pb)

2. Training the age and gender model with age_gender_train.py:

- Data Preparation: Concatenate IMDB-Wiki and Adience datasets
- Organize data into two main folders: age and gender. Each of them should contain train and val subfolders that include images for training and validation respectively.
- Correct age mistakes manually in IMDB-Wiki and utilize the midpoint of age ranges in Adience as ground truth labels (not sure about this)
- Adjust Paths and Labels : Ensure paths in age_gender_train.py are correctly pointing to the data.
- Run the Training Script: Execute age_gender_train.py 

Is my understanding correct?  
Also another issue I am facing is - I haven't been able to download VGGFace2 Dataset. While I came across VGGFace2 HQ repo so I have downloaded these - https://github.com/NNNNAI/VGGFace2-HQ but would training on these give a different model backbone model?
Thank you so much in advance!!

**Comment by mingrongchen** [comment]
我已收到您的邮件，谢谢！
